src/ui/hooks/useScrollToAnchor.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react';
+
+function useScrollToAnchor(): void {
```

Comment:
I'd like to see this unit tested but it doesn't look like it will be straight forward. Happy to see it go in as is seeing as e've manually tested it.